test/bdd/pkg/introduce/introduce_sdk_steps.go

```diff
@@ -219,7 +219,7 @@ func (a *SDKSteps) checkAndStop(agentID, introduceeID string) error {
 			return fmt.Errorf("%q doesn't want to know %q but got %q", agentID, introduceeID, proposal.To.Name)
 		}
 
-		e.Stop(errors.New("stop the protocol"))
+		e.Stop(nil)
```

Comment:
I’m not a fan of nil arguments.

Reply:
This is a way to hide an error in the BDD tests. Stopping the protocol without passing an error prevents logging an error you asked me before.

Reply:
If you don't like it, it's ok. I can create a user error type. It will be much better.

Reply:
Done!